waiter/src/waiter/statsd.clj

```diff
@@ -156,6 +173,7 @@
             config-cluster (:cluster config-map)
             config-server (:server config-map)
             config-publish-interval-ms (:publish-interval-ms config-map)
+            config-refresh-interval-ms (get config-map :refresh-interval-ms 600000)
```

Comment:
Nitpick: Prefer using `(-> 10 from-minutes to-millis)` (or whatever those time functions are called).